os/arch/arm/src/armv7-a/arm_gicv2.c

```diff
@@ -76,59 +76,56 @@
 
 void arm_gic0_initialize(void)
 {
-  unsigned int nlines = arm_gic_nlines();
-  unsigned int irq;
-
-  arm_gic_dump("Entry arm_gic0_initialize", true, 0);
-
-  /* Initialize SPIs.  The following should be done only by CPU0. */
-
-  /* A processor in Secure State sets:
-   *
-   * 1. Which interrupts are non-secure (ICDISR).  All set to zero (group
-   *    0).
-   * 2. Trigger mode of the SPI (ICDICFR). All fields set to 0b01->Level
-   *    sensitive, 1-N model.
-   * 3. Interrupt Clear-Enable (ICDICER)
-   * 3. Priority of the SPI using the priority set register (ICDIPR).
-   *    Priority values are 8-bit unsigned binary. A GIC supports a
-   *    minimum of 16 and a maximum of 256 priority levels. Here all
-   *    are set to the middle priority 128 (0x80).
-   * 4. Target that receives the SPI interrupt (ICDIPTR).  Set all to
-   *    CPU0.
-   */
-
-  /* Registers with 1-bit per interrupt */
-
-  for (irq = GIC_IRQ_SPI; irq < nlines; irq += 32)
-    {
-      putreg32(0x00000000, GIC_ICDISR(irq));   /* SPIs group 0 */
-      putreg32(0xffffffff, GIC_ICDICER(irq));  /* SPIs disabled */
-    }
-
-  /* Registers with 2-bits per interrupt */
-
-  for (irq = GIC_IRQ_SPI; irq < nlines; irq += 16)
-    {
-      putreg32(0x55555555, GIC_ICDICFR(irq));  /* SPIs level sensitive */
-    }
-
-  /* Registers with 8-bits per interrupt */
-
-  for (irq = GIC_IRQ_SPI; irq < nlines; irq += 4)
-    {
-      putreg32(0x80808080, GIC_ICDIPR(irq));   /* SPI priority */
-      putreg32(0x01010101, GIC_ICDIPTR(irq));  /* SPI on CPU0 */
-    }
+	unsigned int nlines = arm_gic_nlines();
```

Comment:
This changs is seen to not related to PM.
Please leave this out of the commit

Reply:
Removed~

Reply:
Hi @ewoodev ,

Please help to check this PR: https://github.com/Samsung/TizenRT/pull/6025

Thank you~